KerbalStuff/blueprints/profile.py

```diff
@@ -20,10 +21,10 @@ def view_profile(username: str) -> str:
         abort(404)
     if not user.public:
         if not current_user:
-            abort(401)
+            abort(403)
```

Comment:
This seems like a place to keep 401. The problem that we're reporting is that the user is not authenticated at all, not that their specific user lacks access, which sounds like what Unauthorized is for to me.